### ISSUE TITLE
DataTable Skeleton - headers prop

### ISSUE THREAD

**Issue by ispyinternet** [IBM/carbon-components-svelte#413]
Wondering why it would be different to the normal datatable?

Given you already have the headers setup for the DataTable, you have to do this for the skeleton:

```
<DataTableSkeleton headers="{headers.map(h=>h.value)}" />
```

Not a big deal granted. But?

**Comment by metonym** [comment]
Agreed – we can expand the type to be consistent with the DataTable

**Comment by metonym** [comment]
Released in v0.24.

Now you can share `headers` between DataTable and DataTableSkeleton:

```svelte
{#if loading}
  <DataTableSkeleton {headers} />
{:else}
  <DataTable {headers} {rows} />
{/if}
```
